magento/magento-cloud-patches
# MCLOUD-5930: BUNDLE-2554: [Amazon Pay] Unable to switch credit card during checkout

Fix for BUNDLE-2554 - Amzon-pay VBE module (Unable to switch credit card during checkout)

Description:

Fix for issue in Amazon-pay bundled extension in which error message is thrown when attempting to change payment card during amazon checkout flow


Fixed Issues (if relevant):
https://jira.corp.magento.com/browse/MCLOUD-5930
https://jira.corp.magento.com/browse/BUNDLE-2554

Manual testing scenarios
Ensure that customer can change from one saved payment card to another without error during amazon checkout flow

Negative test scenario

N/A

Contribution checklist
 Pull request has a meaningful description of its purpose
 All commits are accompanied by meaningful commit messages
 All new or changed code is covered with unit/integration tests (if applicable)
 All automated tests passed successfully (all builds on Travis CI are green)

releasenotes
Fixes an issue with the Amazon Pay payment widget that prevented customers from changing the payment method on the _Review & Payments_ step during the checkout process.
